src/main/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerDelegate.scala

```diff
@@ -48,7 +48,7 @@ private[tracker] class InstanceTrackerDelegate(
   // TODO(jdef) support pods when counting launched instances
   override def countActiveSpecInstances(appId: PathId): Future[Int] = {
     import scala.concurrent.ExecutionContext.Implicits.global
-    instancesBySpec().map(_.specInstances(appId).count(instance => instance.isActive || instance.isReserved))
+    instancesBySpec().map(_.specInstances(appId).count(instance => instance.isActive))
```

Comment:
Same as above: `Created` (the instance after the match before we get any status update) was considered active while `Provisioned` is not.